habitat_baselines/rl/ddppo/ddp_utils.py

```diff
@@ -182,6 +182,9 @@ def load_resume_state(filename_or_config: Union[Config, str]) -> Optional[Any]:
     if not osp.exists(filename):
         return None
 
+    print("error: attempt to load resume state from ", filename)
+    exit(0)
```

Comment:
Why is this change here? Should this be part of the main repo.
Why not raise an error instead of printing and exiting?